scripts/sct_label_vertebrae.py

```diff
@@ -39,20 +38,19 @@
 # PARAMETERS
 class Param:
     ## The constructor
-    def __init__(self):
         # self.path_template = path_sct+'/data/template/'
-        self.shift_AP_brainstem = 30
-        self.size_AP_brainstem = 11#15
-        self.shift_IS_brainstem = 15#15
-        self.size_IS_brainstem = 30#30
-        self.size_RL_brainstem = 1#
-        self.shift_AP = 32#0#32  # shift the centerline towards the spine (in voxel).
-        self.size_AP = 11#41#11  # window size in AP direction (=y) (in voxel)
-        self.size_RL = 1 #1 # window size in RL direction (=x) (in voxel)
-        self.size_IS = 19  # window size in IS direction (=z) (in voxel)
-        self.shift_AP_visu = 15#0#15  # shift AP for displaying disc values
-        self.smooth_factor = [3, 1, 1]  # [3, 1, 1]
-        self.fig_anat_straight = 50
+        shift_AP_brainstem = 30
```

Comment:
I also test using class variable, but it would have to many unexpected side effect, I reverted to using  __init__